config/manifests/bases/vault-config-operator.clusterserviceversion.yaml

```diff
@@ -56,11 +56,28 @@ spec:
       kind: KubernetesAuthEngineRole
```

Comment:
can you add this: operatorframework.io/cluster-monitoring: "true" here? It should have been there. In the annotations section

Reply:
Removed.